core/src/test/java/org/elasticsearch/index/engine/ShadowEngineTests.java

```diff
@@ -185,15 +183,10 @@ public long throttleTimeInNanos() {
         return new Store(shardId, EMPTY_SETTINGS, directoryService, new DummyShardLock(shardId));
     }
 
-    protected IndexDeletionPolicy createIndexDeletionPolicy() {
-        return new KeepOnlyLastDeletionPolicy(shardId, EMPTY_SETTINGS);
-    }
-
     protected SnapshotDeletionPolicy createSnapshotDeletionPolicy() {
-        return new SnapshotDeletionPolicy(createIndexDeletionPolicy());
+        return new org.apache.lucene.index.SnapshotDeletionPolicy(new KeepOnlyLastCommitDeletionPolicy());
```

Comment:
This need not be fully qualified right?  (Remove the org.apache.lucene.index. prefix)